docker/Dockerfile

```diff
@@ -53,7 +53,9 @@ RUN apt-get update -qq \
 USER $NB_USER
 RUN pip install --no-cache-dir datalad[full] pydra
 RUN pip install --no-cache-dir https://github.com/dandi/dandi-cli/archive/master.zip
-RUN cd /data && datalad install ///labs/buzsaki/NWB/SenzaiNeuron2017
+RUN cd /data && datalad install ///labs/buzsaki/NWB/SenzaiNeuron2017 \
+    && datalad install -r ///allen-brain-observatory/
```

Comment:
Fwiw, could be done in a single call to install with -r

Reply:
roger! this dockerfile is rather temporary to get things going, will need cleanup and optimization so we can make the container as small as possible.